src/node/CarplayNode.ts

```diff
@@ -62,14 +86,24 @@ export default class CarplayWS extends EventEmitter {
   }
 
   start = async () => {
-    const { knownDevices } = DongleDriver
+    // Find device to "reset" first
+    let device = await this.findDevice()
```

Comment:
Can be a const

Reply:
This has to be a `let` since we'll be replacing the value after resetting and closing with the fresh one we get at line 115